requirements_dev.txt

```diff
@@ -2,3 +2,4 @@ pytest>=7,<8
 pytest-asyncio>=0.14.0
 deepmerge==0.1.1
 flake8==3.8.4
+jmespath==1.0.1
```

Comment:
Needs to be added to requirements in `setup.py` instead. Then it should be lax version specifier (e.g. `jmespath>1.0.0`